src/components/Header/Header.stories.js

```diff
@@ -0,0 +1,110 @@
+import createTheme from '../../styles/theme';
+
+import Header from './index';
+import readme from './README.md';
+
+const stories = storiesOf('Header', module);
+const theme = createTheme();
+
+const Icon = () => {
+  const style = {
+    maxWidth: '25px',
+    paddingTop: '4px',
+  };
+
+  return (
+    <img
+      src="https://cdn3.iconfinder.com/data/icons/google-material-design-icons/48/ic_keyboard_arrow_right_48px-512.png"
+      style={style}
+      alt=""
+      prop=""
+    />
+  );
+};
+
+const HeaderOnlyWithTitle = () => (
+  <ThemeProvider theme={theme}>
+    <Header
+      title={text('title', 'My awesome title')}
+      alignText={select('alignText', {
+        left: 'left',
+        center: 'center',
+        right: 'right',
+      }, 'left')}
+    />
+  </ThemeProvider>
+);
+
+const HeaderWithSubTitle = () => (
+  <ThemeProvider theme={theme}>
+    <Header
+      alignText={select('alignText', {
+        left: 'left',
+        center: 'center',
+        right: 'right',
+      }, 'left')}
+      title={text('title', 'My awesome title')}
+      subtitle={text('subtitle', 'hoooray!')}
+    />
+  </ThemeProvider>
+);
+
+const HeaderWithThumbnail = () => (
+  <ThemeProvider theme={theme}>
+    <Header
+      alignText={select('alignText', {
+        left: 'left',
+        center: 'center',
+        right: 'right',
+      }, 'left')}
+      title={text('title', 'My awesome title')}
+      subtitle={text('subtitle', 'hoooray!')}
+      thumbnail={text('thumbnail', 'https://camo.githubusercontent.com/82055944adc1659d8435e6d0187eaf71485f4d2d/687474703a2f2f6d656d65642e636f6d2e62722f7265736f75726365732f696d672f6c6f676f2e706e67')}
+    />
+  </ThemeProvider>
+);
+
+const HeaderWithIcon = () => (
+  <ThemeProvider theme={theme}>
+    <Header
+      alignText={select('alignText', {
+        left: 'left',
+        center: 'center',
+        right: 'right',
+      }, 'left')}
+      title={text('title', 'My awesome title')}
+      subtitle={text('subtitle', 'hoooray!')}
+      thumbnail={text('thumbnail', 'https://camo.githubusercontent.com/82055944adc1659d8435e6d0187eaf71485f4d2d/687474703a2f2f6d656d65642e636f6d2e62722f7265736f75726365732f696d672f6c6f676f2e706e67')}
+      icon={
+        <Icon />
+      }
+    />
+  </ThemeProvider>
+);
+
+const HeaderTitleWithDivider = () => (
+  <ThemeProvider theme={theme}>
+    <Header
+      alignText={select('alignText', {
+        left: 'left',
+        center: 'center',
+        right: 'right',
+      }, 'left')}
+      divider
+      dividerColor={color('dividerColor', '#20afd6')}
+      title={text('title', 'My awesome title')}
+    />
+  </ThemeProvider>
+);
+
+stories
+  .add('only with title', withDocs(readme, HeaderOnlyWithTitle))
+  .add('with subtitle', withDocs(readme, HeaderWithSubTitle))
+  .add('with thumbnail', withDocs(readme, HeaderWithThumbnail))
+  .add('with icon', withDocs(readme, HeaderWithIcon))
+  .add('with divider title', withDocs(readme, HeaderTitleWithDivider));
```

Comment:
Instead of using `withDocs` on each story, you could use the `addDecorator` before start adding the stories.

`javascript
stories
  .addDecorator(withDocs(readme))
  .add('story1', () => <Story1 />)
  .add('story2', () => <Story2 />)
`